Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM phusion/baseimage:0.11
+
+RUN curl -o /tmp/node_setup.sh "https://deb.nodesource.com/setup_11.x"
+RUN bash /tmp/node_setup.sh
+RUN curl -sL https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
+RUN echo 'deb https://dl.yarnpkg.com/debian/ stable main' | tee /etc/apt/sources.list.d/yarn.list
+RUN apt-get update -qq && apt-get install -y \
+    jq \
+    nodejs
+
+
+# contract-helper
+COPY . /near-contract-helper/
+WORKDIR /near-contract-helper
+RUN npm install
```

Comment:
this doesn't have to block, but let's start switching to `yarn` everywhere. Looks like it's faster and more reliable as well.

Reply:
Okay, let's do it after we have the first version.

Reply:
Up to you, it might speed up your testing process )